src/js/controller/controller.js

```diff
@@ -511,6 +511,10 @@ Object.assign(Controller.prototype, {
                     });
                 }
 
+                if (_this.getMute()) {
```

Comment:
We still need the flag. If we do `_model.set('captionsIndex', 1)` here, it does not get applied to the 2nd playlist item in case the first one does not have captions. The `captionsIndex` on model is set to 1, but it is being replaced when the `itemReady` event fires for the next playlist item, that results the `captionsIndex` to be set to 0 again: https://github.com/jwplayer/jwplayer/pull/3269/files#diff-7c257d0959b87bf538adcbf0e4fa5504R119

Reply:
Can we shift the flag to the model?

Reply:
What is the config option set by the ab test? Use that.